libgalois/src/PropertyGraph.cpp

```diff
@@ -6,6 +6,8 @@
 #include <memory>
 #include <utility>
 
+#include <arrow/array.h>
```

Comment:
Is this needed?

Reply:
Weirdly, it seems to be the home for the schema and field apis.

Reply:
Ok, then.